lib/suspenders/generators/placeholders_generator.rb

```diff
@@ -0,0 +1,30 @@
+require_relative "base"
+
+module Suspenders
+  class PlaceholdersGenerator < Generators::Base
+    def setup_default_directories
+      DIRS.each { |dir| empty_directory_with_keep_file dir }
+    end
+
+    private
+
+    DIRS = %w[
+      app/views/pages
+      spec/lib
+      spec/controllers
+      spec/helpers
+      spec/support/matchers
+      spec/support/mixins
+      spec/support/shared_examples
+    ].map(&:freeze).freeze
```

Comment:
How do you feel about the `# frozen_string_literal: true` magic comment instead of mapping over these? The double freeze took me a moment to make sense of.

Reply:
Oh right! Thank you for the reminder.